docs/adapters.md

```diff
@@ -14,8 +14,8 @@ The framework can communicate with both source and sink adapters via a protocol
   - [PUB/SUB Explanation](#pub/sub-explanation)
 - [Source Adapters](#source-adapters)
   - [The Image File Source Adapter](#the-image-file-source-adapter)
-- [The Video File Source Adapter](#the-video-file-source-adapter)
-- [Example](#example)
+  - [The Video File Source Adapter](#the-video-file-source-adapter)
+  - [The Looped Video File Source Adapter](#the-looped-video-file-source-adapter)
```

Comment:
Please change to Loop Video File Source Adapter, don't use the 'looped' eveywhere.

Reply:
We could use Cycled Video, but Looped Video is a weird thing.

Reply:
Even better to change to "Video Loop Source Adapter".